ansible/roles/products/common/docker-container/defaults/main.yml

```diff
@@ -16,3 +16,4 @@ container_exists: false
 container_running: false
 setup_directory: ""
 setup_script: ""
+docker_version: docker
```

Comment:
please rename variable to `docker_executable` or something like this.

Reply:
Fixed in the latest commit